concepts/observability-strategy/configurable-monitoring/04-monitoring-api.md

```diff
@@ -7,23 +7,43 @@ apiVersion: telemetry.kyma-project.io/v1alpha1
 metadata:
   name: PrometheusRemoteWrite
 spec:
-  receiver: # singular, different receivers need different pipelines and exporters configs
-    type: otlp # otlp  | custom, default is otlp
-    otlp: # maps to central otlp receiver
+  input: # Metrics can be received from different receivers (OTLP and Prometheus), metrics are selected based on labels
```

Comment:
If the code accepts full stop (or semicolon) instead of comma, that would look nicer

```suggestion
  input: # Metrics can be received from different receivers (OTLP and Prometheus). Metrics are selected based on labels
```